clients/native/src/client/config/template.rs

```diff
@@ -42,6 +42,12 @@ public_encryption_key_file = '{{ client.public_encryption_key_file }}'
 # sent but not received back.
 reply_encryption_key_store_path = '{{ client.reply_encryption_key_store_path }}'
 
+# Ethereum private key.
+eth_private_key = '{{ client.eth_private_key }}'
```

Comment:
Why would you try that? It would be a different binary, with a different release version. When we'll switch from this version to the coconut version, an `update` will probably need to be written.

Reply:
What I meant is that this particular template is used regardless of `coconut` feature status. However, `client.eth_private_key` does not exist in the `coconut` case, hence it will try to include it during config generation and will fail

Reply:
I understand what you mean now. `init`ing and `run`ing with `coconut` enabled works just fine with this template from what I've tested, so we should be good.